File: src/main/java/com/destino/projeto_destino/dto/pacote/PacoteRegistroDTO.java
package com.destino.projeto_destino.dto.pacote;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.ArrayList;
import java.util.UUID;

public record PacoteRegistroDTO(
        String nome,
        String descricao,
        ArrayList<String> tags,
        BigDecimal preco,
        LocalDate inicio,
        LocalDate fim,
        int disponibilidade,
        int transporte,
        int hotel,
        int pacoteFoto,
        UUID funcionario
) {
}

File: src/main/kotlin/com/fatec/destino/dto/pacote/PacoteRegistroDTO.kt
package com.fatec.destino.dto.pacote

import java.math.BigDecimal
import java.time.LocalDate
import java.util.UUID

data class PacoteRegistroDTO(
    val nome: String,
    val descricao: String,
    val tags: List<String>,
    val preco: Int,
    val inicio: LocalDate,
    val fim: LocalDate,
    val disponibilidade: Int,
    val transporte: Long,
    val hotel: Long,
    val pacoteFoto: Long,
    val funcionario : UUID
)
